### PR TITLE
Update data model for operations committee

### PR DESCRIPTION
**Pull request by cthoyt**
Improve the way affiliations are annotated to reduce confusion about what "wikidata" means

### CODE REVIEW

**Comment by erik-whiting** [comment]
@matentzn you told me to check with you before merging PRs for a few weeks. this one looks fine to me although there isn't an issue for it or conversation about why it might be needed. either way, looks good though.

File: _data/operations.yml [hunk 1]
```diff
@@ -1,138 +1,154 @@
...
+- affiliation:
+    name: Kansas State University, Manhattan, KS
+    ror: 05p1j8758
   country: USA
   github: handemcginty
   groups:
   - editorial
   - outreach
-  name: Hande Küçük McGinty
+  name: "Hande K\xFC\xE7\xFCk McGinty"
```
Comment (rays22) [inline _data/operations.yml:146]: This change does not look good.

Comment (cthoyt) [reply _data/operations.yml:146]: oops i forgot to fix the ascii. fixing momentarily

**Comment by matentzn** [comment]
I agree with @rays22 on the character encodings, that seems out of place.

I notice the HTML templates do not use the ROR for anything, but that can be a future issue.

File: _data/operations.yml [hunk 2]
```diff
@@ -248,69 +278,75 @@ members:
...
   github: shawntanzk
   groups:
   - technical
   name: Shawn Tan
   orcid: 0000-0001-7258-9596
-  ror: 02catss52
   wikidata: Q57023310
-- affiliation: Université de Sherbrooke, Québec
+- affiliation:
+    name: "Universit\xE9 de Sherbrooke, Qu\xE9bec"
```
[the remark this answers is not in the document]
Comment (cthoyt) [reply _data/operations.yml]: Fixed in 77e53269

File: _data/operations.yml (hunk 1 above)
Comment (cthoyt) [reply _data/operations.yml]: Fixed in 77e53269

**Review by rays22 — APPROVED** [review]

**Pull request merged**